src/routes/cartRouter.ts

```diff
@@ -62,4 +63,7 @@ router.get(
   isCartIdExist,
   cartControllers.buyerCheckout
   );
-export default router;
\ No newline at end of file
+
+router.post("/user-get-order-status",userAuthorization(["buyer"]),validation(orderStatusSchema),isOrderExist, cartControllers.buyerGetOrderStatus )
```

Comment:
Instead of sending orderId into the body send it into the params, And use GET method instead of POST method.